### PR TITLE
2026au

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/cypress/api_measure_evaluator.rb [hunk 1]
```diff
@@ -489,9 +489,9 @@ def detypo_zip(file_name)
       patient_scoop_and_filter = Cypress::ScoopAndFilter.new(pt.measures)
       Zip::ZipOutputStream.open("tmp/#{file_name}_dedupe.zip") do |z|
         patient_telecoms.each_value do |entry_hash|
-          options = { ry2026_submission: pt.bundle.major_version == '2025', submission_program: pt.submission_program,
-                      start_time: pt.start_date, end_time: pt.end_date, patient_telecoms: [entry_hash[:phone_number]],
-                      medicare_beneficiary_identifier: entry_hash[:mbi] }
+          options = { ry2026_submission: pt.bundle.major_version == '2025', ry2027_submission: pt.bundle.major_version == '2026',
```
Comment (elsaperelli) [inline lib/cypress/api_measure_evaluator.rb:492]: What about updates to lines 610-612?

**Review by elsaperelli — CHANGES_REQUESTED** [review]
One comment so far- but looking good!

**Review by elsaperelli — APPROVED** [review]
🥳
